lol/src/chessboard.js

```diff
@@ -75,5 +75,16 @@ function dragEnd(event){
    // console.log(event.currentTarget);
 } 
 
+function dragEnter(event){
+    event.preventDefault();
+       event.currentTarget.classList.add('over');
```

Comment:
currentTarget의 경우 event listener가 붙어 있었던 쪽에 이벤트가 실행되기 때문에 현재 target으로 해주는게 맞는 것 같아요

```suggestion
       event.target.classList.add('over');
```

Reply:
감사합니다!